packages/offix-datastore/src/replication/GraphQLCRUDReplicator.ts

```diff
@@ -28,12 +33,23 @@ export function convertPredicateToFilter(predicate: PredicateFunction): any {
  * Performs replication using GraphQL
  */
 export class GraphQLCRUDReplicator implements IReplicator {
+
   private client: Client;
-  private queries: Map<string, GraphQLDocuments>;
+  // private networkStatus: NetworkStatus
```

Comment:
Ignore that class completely and everything that is in the replicator. 
I have some 2 different sets of changes for this that I'm exploring.